src/utils.js

```diff
@@ -79,16 +79,19 @@ module.exports = {
 		return new Promise((resolve, reject) => {
 			function getResponse() {
 				// Present the user with a prompt for input.
-				res.reply(prompt);
+				robot.emit('ibmcloud.formatter', { response: res, message: prompt});
+
 
 				// Control the response when the timeout expires.
 				dialog.dialogTimeout = function(msg) {
 					/* istanbul ignore next */
-					res.reply(i18n.__('conversation.timed.out'));
+					robot.emit('ibmcloud.formatter', { response: res, message: i18n.__('conversation.timed.out')});
 				};
 
 				// Handle the expected response.
 				dialog.addChoice(regex, (msg) => {
+					msg.match[1] = that.stripBotName(robot.name, msg.match[1]);
```

Comment:
This line should be removed, since the match will only include what is matched by the regex. If the user purposefully wants to match the botname, this will remove it. Also its not always the case that `match[1]` will exist.

for example, the user inputs "hubot Y"

regex = `/Y/i`
match = [ 'Y', index: 6, input: 'hubot Y' ]

regex = `/(Y)/i`
match = [ 'Y', 'Y', index: 6, input: 'hubot Y' ]

regex = `/.*/i`
match = [ 'hubot Y', index: 0, input: 'hubot Y' ]

So if a capturing group is not used in the regex, it will crash.